Meadowlark-js/backends/meadowlark-mongodb-backend/src/BackendFacade.ts

```diff
@@ -40,3 +42,7 @@ export async function updateDocumentById(request: UpdateRequest): Promise<Update
 export async function securityMiddleware(middlewareModel: MiddlewareModel): Promise<MiddlewareModel> {
   return SecurityMiddleware.securityMiddleware(middlewareModel, await getSharedClient());
 }
+
+export async function createAuthorizationClient(request: CreateClientRequest): Promise<CreateClientResult> {
```

Comment:
Using "Authorization" in the name here because our backends already use "Client" to mean DB client.

Reply:
Understand about difficulty with the term "client", but I'd like to keep the "authorization" term consistent throughout. How about 'createAuthorizationClientDocument'? The key difference from a DB client is that this is a document stored in the DB.

Reply:
@bradbanister This works as well